components/TaskContainer.vue

```diff
@@ -3,7 +3,14 @@
     <div class="title-text">{{ $t('PageTitle') }}</div>
     <div class="space-between flex-box margin-top-28 margin-bottom-34">
       <div>
-        <button @click="showAddTodoCard()">{{ $t('create') }}</button>
+        <button
+          class="create-button text-button flex-box"
+          :disabled="isSearching"
+          @click="showAddTodoCard()"
```

Comment:
Can we remove the function invocation here?

```suggestion
          @click="showAddTodoCard"
```

Reply:
resolved